contributing/development/configuring_an_ide/visual_studio_code.rst

```diff
@@ -154,7 +154,7 @@ To run and debug the project you need to create a new configuration in the ``lau
 
 The name under ``program`` depends on your build configuration,
 e.g. ``godot.linuxbsd.editor.dev.x86_64`` for 64-bit LinuxBSD platform with
-``platform=editor`` and ``dev_build=yes``.
+``platform=linuxbsd`` and ``dev_build=yes``.
```

Comment:
I'm not sure this is correct, the platform part is automatic, I think it should instead be:
```suggestion
``target=editor`` and ``dev_build=yes``.
```
That's the option that's *needed*, the platform is automatic unless you're cross compiling